2 - RTB/Documentazione interna/Norme di Progetto/Norme di Progetto.typ

```diff
@@ -769,91 +750,109 @@ Documenti è organizzata in modo da suddividere la documentazione necessaria all
++ *merge non confermato*: qualora a termine della compilazione del documento non venisse confermato il merge da parte del verificatore, significa che è stato individuato un ulteriore errore o correzione da dover apportare al documento prima della sua pubblicazione sul ramo `main` del repository. In questa circostanza sarà dunque necessario rilasciare un'ulteriore review. L'esecuzione riprende dal punto 4.
 
 
 L'azione manuale si riduce solamente al rilascio di review e conferma di merge, mentre tutte le altre attività vengono automatizzate.
-All'approvazione della pull request, e alla conseguente chiusura del branch, un ulteriore automazione integrata su Jira, permette di aggiornare in automatico lo stato della task collegata alla pull request, portandola allo stato di "Completata".
+All'approvazione della pull request, e alla conseguente chiusura del branch, un'ulteriore automazione integrata su Jira, permette di aggiornare in automatico lo stato del task collegato alla pull request, portandolo allo stato di "Completato".
 
-==== Jira
-Jira è lo strumento centrale per la gestione e la tracciabilità delle task assegnate ai membri del gruppo.
-L'integrazione con GitHub permette a Jira di lavorare e apportare modifiche direttamente alla repository del gruppo, permettendo la creazione, gestione e chiusura di branch e al conseguente aggiornamento dello stato delle task.
-Ogni task è identificata da un codice univoco incrementale nel formato `DOC-XXX`, dove `XXX` è un numero positivo sequenziale, che permette di identificarla. Ogni task è caratterizzata da:
+====== Typst
+Il gruppo utilizza Typst come strumento di scrittura e compilazione dei documenti. \
+Al fine di dare una struttura comune ai documenti si è creato un file Template.typ parametrizzato, sfruttando la possibilità di produrre un file PDF compilando insieme più file Typst. Questo file contiene le impostazioni di base per la creazione di un documento:
+- `title`: titolo del documento;
+- `subTitle`: sottotitolo del documento;
+- `docType`: tipologia del documento (Verbale, Documento);
+- `date`: data di creazione del documento;
+- `externalPartecipants`: partecipanti esterni al gruppo;
+- `authors`: autori del documento;
+- `reviewers`: revisori del documento;
+- `missingMembers`: membri assenti durante i meeting;
+- `location`: luogo di incontro;
+- `timeStart`: ora di inizio incontro;
+- `timeEnd`: ora di fine incontro;
+- `showLog`: flag che indica se mostrare il tracciamento delle modifiche;
+- `showIndex`: flag che indica se mostrare l'indice;
+- `isExternalUse`: flag che indica se il documento è per uso esterno.
+Al momento della creazione di un nuovo documento sarà sufficiente importare il modello e specificare i parametri sopra elencati. \
+Al fine di semplificare la procedura di creazione di un documento, è stato condiviso un documento di testo denominato quickstart.txt che contiene la configurazione base per la stesura dei documenti.
```

Comment:
"quickstart.txt" lo metterei tra virgolette

Reply:
non sarebbe meglio in italic come con gli altri file?

Reply:
hai ragione, metti italic